pkg/webservice/state_dump.go

```diff
@@ -170,16 +186,21 @@ func startBackGroundStateDump(period time.Duration) error {
 
 	if periodicStateDump {
 		var errMsg = "state dump already running"
-		log.Logger().Error(errMsg)
+		yunikornLog.Logger().Error(errMsg)
 		return fmt.Errorf(errMsg)
 	}
 
+	stateDumpFilePath, exists := os.LookupEnv(stateDumpFilePathEnvKey)
+	if !exists {
+		stateDumpFilePath = defaultStateDumpFilePath
+	}
```

Comment:
We should avoid dup code, this is the same as 166 - 169
if this is for dynamic reloading new ENV, can we have a convenient function `getStateDumpFilePath()`